### DISCUSSION TITLE
AntD 的 ConfigProvider 对 FormDialog/FormDrawer 不生效

### DISCUSSION THREAD

**Discussion by anhulife** [alibaba/formily#3298] [Q&A 2.x]
如题，想通过 ConfigProvider 来更改样式前缀，目前发现除了 FormDialog/FormDrawer 不生效，其他都生效了。是因为弹出层的渲染方式导致的吗？

```
<ConfigProvider prefixCls="other">
  <App />
</ConfigProvider>
```

**Comment by anhulife** [comment]
可以看到渲然根节点 root 里的 className 是生效的，但是弹出层里的 className 是没生效的。

![root元素内的](https://user-images.githubusercontent.com/970903/181502033-35b2797e-c5bb-42c7-a64f-ad10893372ac.png)
![弹出层内的](https://user-images.githubusercontent.com/970903/181502045-c985c489-94d9-48c7-b179-0550ecd17e5d.png)

**Reply by janryWang** [reply]
需要用FormDialog.Portal共享context
